packages/actor-tests/bedrock/deposit-eth.ts

```diff
@@ -0,0 +1,88 @@
+import * as fs from 'fs'
+
+import { utils, Wallet, providers, constants } from 'ethers'
+import {
+  CrossChainMessenger,
+  ETHBridgeAdapter,
+  StandardBridgeAdapter,
+} from '@eth-optimism/sdk'
+import { predeploys } from '@eth-optimism/contracts-bedrock'
+import { sleep } from '@eth-optimism/core-utils'
+
+import { actor, setupActor, run } from '../lib/convenience'
+
+interface Context {
+  wallet: Wallet
+}
+
+actor('Dev account sender', () => {
+  let l1Provider: providers.JsonRpcProvider
+
+  let l2Provider: providers.JsonRpcProvider
+
+  let wallet: Wallet
+
+  let messenger: CrossChainMessenger
+
+  let contracts: any
+
+  setupActor(async () => {
+    l1Provider = new providers.JsonRpcProvider(process.env.L1_RPC)
+    l2Provider = new providers.JsonRpcProvider(process.env.L2_RPC)
+    wallet = new Wallet(process.env.PRIVATE_KEY)
+    contracts = JSON.parse(
+      fs.readFileSync(process.env.CONTRACTS_JSON_PATH).toString('utf-8')
```

Comment:
You can `require` a json file and its automatically turned into an object, could reduce some boilerplate